docker_entrypoint.sh

```diff
@@ -32,4 +55,12 @@ else
   fi
 fi
 
-exec ./jellyfin/jellyfin --datadir /config --cachedir /cache --ffmpeg /usr/bin/ffmpeg
+# Run Jellyfin with appropriate options based on the architecture
+if [ "$ARCH" = "aarch64" ]; then
+  exec ./jellyfin/jellyfin --datadir /config --cachedir /cache --ffmpeg /usr/bin/ffmpeg
+elif [ "$ARCH" = "x86_64" ]; then
+  exec ./jellyfin/jellyfin --datadir /config --cachedir /cache --ffmpeg /usr/lib/jellyfin-ffmpeg/ffmpeg
+else
+  echo "Unsupported architecture: $ARCH"
```

Comment:
my mac says `arm64` for `uname -a`. what systems did you test this build on?

Reply:
Initially I had used `arm64` in this conditional, but when I ran the s9pk on my Pi testbox, the logs were showing the error "Unsupported architecture aarch64" - So I replaced `arm64` with `aarch64`.

My understanding was arm64 and aarch64 were functionally the same, but if I was mistaken in this understanding, I can update the conditional to include the appropriate `exec` command for `arm64` in addition to `aarch64`

Reply:
I would just sidestep this issue entirely by building the x86 and arm builds the same way